botnim/sync.py

```diff
@@ -52,121 +52,110 @@ def openapi_to_tools(openapi_spec):
     return ret
 
 def update_assistant(config, config_dir, production, replace_context=False):
-    tool_resources = None
-    tools = None
-    print(f'Updating assistant: {config["name"]}')
-    # Load context, if necessary
-    if config.get('context'):
-        for context_ in config['context']:
-            name = context_['name']
-            if not production:
-                name += ' - פיתוח'
-            vector_store = client.beta.vector_stores.list()
-            vector_store_id = None
-            for vs in vector_store:
-                if vs.name == name:
-                    if replace_context:
-                        client.beta.vector_stores.delete(vs.id)
-                    else:
-                        vector_store_id = vs.id
-                    break
-            if vector_store_id is None:
-                if 'files' in context_:
-                    files = list(config_dir.glob(context_['files']))
-                    existing_files = client.files.list()
-                    # delete existing files:
-                    for f in files:
-                        for ef in existing_files:
-                            if ef.filename == f.name:
-                                client.files.delete(ef.id)
-                    file_streams = [f.open('rb') for f in files]
-                elif 'split' in context_:
```

Comment:
See here the 'split' functionality - reading the source file and splitting it to bits that would be considered individual files.

Reply:
yes, so I think the 'split' should be a true/false flag instead of serving two different purposes.